### ISSUE TITLE
Definition for rule 'no-loops' was not found

### ISSUE THREAD

**Issue by coryhouse** [buildo/eslint-plugin-no-loops#1]
This isn't working for me. I've installed and added to the array of plugins in .eslintrc. 

If I add a "no-loops" rule to my .eslintrc (unclear if this is necessary), I get the following:

`1:1  warning  Definition for rule 'no-loops' was not found  no-loops`

Am I missing something?

**Comment by gabro** [comment]
Hi @coryhouse!
Plugin rules need to be prefixed with the plugin name.

This is slightly odd for plugins defining a single rule like this one, but you should do something like:

``` json
{
  "plugins": [
    "no-loops"
  ],
  "rules": {
    "no-loops/no-loops": 2
  }
}
```

That having being said, since this plugin provides a single rule, it is also active by default, so you can add the plugin and be done with it:

``` json
{
  "plugins": [
    "no-loops"
  ]
}
```

Hope it helps

**Comment by coryhouse** [comment]
Thanks! Adding the rule as you suggested got it running. 

However, the default isn't applying for me. This rule only runs if I add "no-loops/no-loops": 2 to my .eslintrc. Merely adding "no-loops" to the array of plugins does nothing.

**Comment by gabro** [comment]
I'm glad at least you got this working! The behavior you're seeing is odd. This https://github.com/buildo/eslint-plugin-no-loops/blob/master/index.js#L5-L7 should enable the rule by default, according to the eslint documentation.

Which version of eslint are you using?

**Comment by coryhouse** [comment]
I'm running ESLint 2.2.0 via babel-eslint 5.0.0 using Webpack and eslint-loader 1.3.0.

Perhaps eslint-loader doesn't honor the setting.

**Comment by gabro** [comment]
I finally had some time to look into this, and apparently the `ruleConfig` option is not supported any longer with recent versions of eslint (>= 2.0.0, looks like). https://github.com/eslint/eslint/issues/4952

I could ship a recommended config with the plugin, but since I'm providing a single rule there's really no point. I'll update the README, thanks for reporting this, @coryhouse!

**Comment by gabro** [comment]
Done, and published v0.2.0 on npm!
